src/cdomain/value/cdomains/intDomain.ml

```diff
@@ -572,6 +572,14 @@ module IntervalArith (Ints_t : IntOps.IntOps) = struct
     let min_ik' = Ints_t.to_bigint min_ik in
     let t = List.find_opt (fun x -> Z.compare l x >= 0 && Z.compare x min_ik' >= 0) ts in
     BatOption.map_default Ints_t.of_bigint min_ik t
+  let is_upper_threshold u =
+    let ts = if get_interval_threshold_widening_constants () = "comparisons" then WideningThresholds.upper_thresholds () else ResettableLazy.force widening_thresholds in
```

Comment:
Why only in the comparisons configuration and not if they're manually supplied?

Reply:
This is the same logic as how widening thresholds are looked up in the first place. I think it works with both, but the `comparisons` vs `all` have been implemented very differently.